### PR TITLE
Add Restriction for Kibana with Firefox

### PR DESCRIPTION
**Pull request by chaconpiza**
- Filtering more than one element using Firefox is only possible using the Kibana Query
Language in the Filter Line. More info [1]

[1] elastic/kibana#41567

### CODE REVIEW

File: Markdown/cmm-2.0.14/release-notes/2-restrictions.md [hunk 1]
```diff
@@ -47,7 +47,6 @@ project in OpenStack Horizon and then try to access Kibana, the Kibana dashboard
 be not loading properly.
 
 
-### Workaround:
-
-For all browser versions suppported: Please use the default options of your browser to reload the page.
+* Workaround:
+  For all browser versions suppported: Please use the default options of your browser to reload the page.
```
Comment (GoebelL) [inline Markdown/cmm-2.0.14/release-notes/2-restrictions.md:51]: Some typo here - 'suppported' should be  'supported'.

File: Markdown/cmm-2.0.14/release-notes/2-restrictions.md [hunk 2]
```diff
@@ -47,7 +47,17 @@ project in OpenStack Horizon and then try to access Kibana, the Kibana dashboard
 be not loading properly.
 
 
-### Workaround:
+* Workaround:
+  For all browser versions suppported: Please use the default options of your browser to reload the page.
 
-For all browser versions suppported: Please use the default options of your browser to reload the page.
 
+### Filtering more than one element in Kibana using Firefox
```
Comment (GoebelL) [inline Markdown/cmm-2.0.14/release-notes/2-restrictions.md:54]: Use capitalized heading style, should be
### Filtering More Than One Element in Kibana Using Firefox

https://www.webucator.com/how-to/how-capitalize-headings-titles.cfm

**Review by GoebelL — APPROVED** [review]

File: Markdown/cmm-2.0.14/release-notes/2-restrictions.md (hunk 1 above)
Comment (chaconpiza) [reply Markdown/cmm-2.0.14/release-notes/2-restrictions.md]: Done

**Review by GoebelL — APPROVED** [review]

File: Markdown/cmm-2.0.14/release-notes/2-restrictions.md (hunk 2 above)
Comment (chaconpiza) [reply Markdown/cmm-2.0.14/release-notes/2-restrictions.md]: Done

**Pull request merged**